### DISCUSSION TITLE
cron "update labels"

### DISCUSSION THREAD

**Discussion by saba1996** [YetiForceCompany/YetiForceCRM#16813] [General]
Hello, if I change the time, it still starts every 10 minutes. It doesn't complete, it was working at first and then it started showing a similar error.

version 6.1.0 

![image](https://user-images.githubusercontent.com/56167014/212629718-4d02241e-e64c-4c81-a4b0-fd92a354ba61.png)

**Comment by mariuszkrzaczkowski** [comment]
is that a question or are you reporting a bug?

**Reply by saba1996** [reply]
question
